cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -2141,28 +2141,39 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
         }
 
         Opcode::FcvtFromSint => {
-            let (ext_spec, src_size) = match ctx.input_ty(insn, 0) {
-                types::I8 | types::I16 => (Some(ExtSpec::SignExtendTo32), OperandSize::Size32),
-                types::I32 => (None, OperandSize::Size32),
-                types::I64 => (None, OperandSize::Size64),
-                _ => unreachable!(),
-            };
+            let output_ty = ty.unwrap();
+            if !output_ty.is_vector() {
+                let (ext_spec, src_size) = match ctx.input_ty(insn, 0) {
+                    types::I8 | types::I16 => (Some(ExtSpec::SignExtendTo32), OperandSize::Size32),
+                    types::I32 => (None, OperandSize::Size32),
+                    types::I64 => (None, OperandSize::Size64),
+                    _ => unreachable!(),
+                };
 
-            let src = match ext_spec {
-                Some(ext_spec) => RegMem::reg(extend_input_to_reg(ctx, inputs[0], ext_spec)),
-                None => input_to_reg_mem(ctx, inputs[0]),
-            };
+                let src = match ext_spec {
```

Comment:
These white spaces are curious. I do not see them indicated on the file I push?

Reply:
It's fine: `rustfmt` probably bumped them in a bit since they are now nested inside `if !output_ty.is_vector()`.